src/components/tooltip/styles.ts

```diff
@@ -0,0 +1,132 @@
+import { styled } from '@linaria/react'
+
+export const ElToolTipContainer = styled.div`
```

Comment:
**nit-pick:** Again, I don't think Tip should be capitalised; i.e. `ElTooltipContainer`. Looks like there's a lot of this throughout the PR.